### DISCUSSION TITLE
Databasing | Standardization | caseCase | Vertical Bar Space

### DISCUSSION THREAD

**Discussion by Digitization-RBG-HAM** [Symbiota/community-board#451] [Q&A]
Hello Symbiota,

I’m trying to standardize our data entry across volunteers and employees over time. One perhaps insignificant issue is that of upper or lower case in fields where we require controlled vocabulary vs. verbatim fields.

Example: “Life Stage”. Examples in Darwin Core are all lowercase: (larva, juvenile). Should all fields with controlled vocabulary be in lower case as shown in the Darwin Core help links, or does it not matter?

Also, in most cases, if there are multiple entries for one field (“Associated Taxa”) there is a best practice in some but not all fields of separating values with the “vertical bar space” ( | ). Should we do this with all controlled vocabulary fields with multiple values?

The answers to these questions will also help for future database managers who will need to standardize fields and language across the database, easing database management and batch transfers.

Thanks for your help.

**Comment by Barkworth** [comment]
Switching to using | instead of commas at this stage seems theoretically desirable but what is current practice? Does the DwC generator insert | when appropriate? That *might* be less disruptive than converting existing records and practices.  
Surely, if a site is using a controlled vocabulary, the use of upper and lower case would be defined there. Given that many languages do not have an upper and lower case concept, I would hope it is irrelevant.

**Comment by Digitization-RBG-HAM** [comment]
You pose an excellent question that I'd love to hear an answser to as well: Does the DwC generator insert " | " when appropriate? I'm new to this so I don't know, but that would be a factor in making a decision on how we enter data at the Herbarium.

Re: Languages - while that is an excellent point and observation, from a databasing standpoint, much like the " | " it may be a factor when batch processing or editing, which is part of the reason why I'm asking. While it may not be significant because of the variety of languages, one choice or the other (caps or not to start a sentence) may greatly affect processing tens- to hundreds- of thousands of files.

Since Darwin Core has all lowercase for most entries, the assumption there is to follow their example. But it may not matter - I just want to be 100% sure it doesn't before we start entering all the data and digitizing another 20 thousand specimens :)

**Reply by themerekat** [reply]
The Darwin Core generator in Symbiota doesn't replace any delimiters, so I think the answer is "no" to your first question.

**Comment by Barkworth** [comment]
Completely understood.  There might be a difference for plants? I have never entered life stage, using phenology. Standardization there would be good but I dislike the use of abbreviations. Also I tend to make them active participles. Not good for standardization I know.

**Comment by zygoballus** [comment]
Our collection (Carnegie Museum IZ) uses vertical bars for multiple values ("male | female") and generally lowercase words ("juvenile"), but I don't have strong opinions about either.
